### PR TITLE
Feature/ui feed

### PR DESCRIPTION
**Pull request by zennarchi**

### CODE REVIEW

File: client/src/components/Post/index.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ export default class Post extends React.Component<IProps> {
     }
 
     public componentDidMount(): void {
+        this.setState({page: 1});
```
Comment (didOstap) [inline client/src/components/Post/index.tsx]: тут непотрібний вже setState

File: client/src/components/FeedPost/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import '../Post/style.scss';
+import noAvatar from '../../assets/noAvatar.svg';
+import {Link} from 'react-router-dom';
+
+interface IBody {
+    userId: string;
+    postId: string;
+}
+
+export default class FeedPost extends React.Component<any> {
```
Comment (didOstap) [inline client/src/components/FeedPost/index.tsx]: no need in class component

Comment (didOstap) [inline client/src/components/FeedPost/index.tsx]: Please, use function component

File: client/src/components/Main/Feed.tsx [hunk 3]
```diff
@@ -1,21 +1,53 @@
...
-                <Col>
-                    <h2>Welcome</h2>
-                    <Link to={`/profile/${loggedUsername}`} className='text-danger pl-1'>Profile</Link>
-                    <Link to='/logout' className='text-danger pl-1'>Logout</Link>
-                </Col>
-            </Row>
-        </Container>
-    );
-};
+export class Feed extends React.Component<IProps> {
```
Comment (didOstap) [inline client/src/components/Main/Feed.tsx]: Please, use function component

File: client/src/components/FeedPost/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                <div className='post-header p-2 border'>
+                    <div className='d-flex flex-row'>
+                        <img
+                            src={noAvatar}
+                            alt='avatar'
+                            width={32}
+                            height={32}
+                            className='img-fluid rounded-circle'
+                        />
+                        <Link to='/profile/zennarchi' className='text-dark mt-1 ml-3'>{this.props.author}</Link>
```
Comment (InnaKutsik) [inline client/src/components/FeedPost/index.tsx]: zennarchi?

File: client/src/components/FeedPost/index.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                <div className='description-post pb-3 border-bottom'>
+                    <div className='d-block pl-3 text-description'>
+                        <img
+                            src={noAvatar}
+                            alt='avatar'
+                            width={32}
+                            height={32}
+                            className='img-fluid rounded-circle'
+                        />
+                        <Link to='/profile/zennarchi' className='d-inline-block text-dark ml-2'>
```
Comment (InnaKutsik) [inline client/src/components/FeedPost/index.tsx]: zennarchi?

File: client/src/components/Post/index.tsx (hunk 1 above)
Comment (zennarchi) [reply client/src/components/Post/index.tsx]: Fixed

File: client/src/components/FeedPost/index.tsx (hunk 4 above)
Comment (zennarchi) [reply client/src/components/FeedPost/index.tsx]: Fixed

File: client/src/components/FeedPost/index.tsx (hunk 5 above)
Comment (zennarchi) [reply client/src/components/FeedPost/index.tsx]: Fixed

File: client/src/components/Main/Feed.tsx [hunk 6]
```diff
@@ -1,21 +1,53 @@
...
+                    </Col>
+                    <Col sm={4} className='order-1 order-sm-2 text-sm-center'>
+                        <img
+                            src={noAvatar}
+                            alt='avatar'
+                            width={64}
+                            height={64}
+                            className='img-fluid rounded-circle'
+                        />
+                        <Link to='/profile/zennarchi' className='mt-1 ml-3 text-dark'>zennarchi</Link>
```
Comment (InnaKutsik) [inline client/src/components/Main/Feed.tsx]: zennarchi ?

Comment (zennarchi) [reply client/src/components/Main/Feed.tsx]: Sorry, fixed

**Pull request merged**